packages/perseus/src/widgets/interactive-graphs/mafs-graph.tsx

```diff
@@ -87,12 +88,14 @@ export const MafsGraph = (props: MafsGraphProps) => {
         fullGraphAriaLabel,
         fullGraphAriaDescription,
     } = props;
+    const {type} = state;
     const [width, height] = props.box;
     const tickStep = props.step as vec.Vector2;
 
     const uniqueId = React.useId();
     const descriptionId = `interactive-graph-description-${uniqueId}`;
     const graphRef = React.useRef<HTMLElement>(null);
+    const dependencies = useDependencies();
```

Comment:
A slight stylistic improvement that might make it easier for future readers to see what parts of `dependencies` are being used:

```suggestion
    const {analytics} = useDependencies();
```